wordcards.py

```diff
@@ -26,11 +25,43 @@ def ask_back_words(cards):
             print("Wrong")
 
 
+def make_cards_clean_to_write_in_a_file(cards):
+    clean_cards = list(cards.items())
+    return clean_cards
+
+
+def write_words_to_file(cards, file_name):
+    clean_cards = make_cards_clean_to_write_in_a_file(cards)
+    with open(file_name, "w") as file:
+
+        for item in clean_cards:
+            file.write(str(item[0] + "-" + item[1] + ";"))
```

Comment:
This step where the data format is changed to something that can be written to a file is called serialize. Usually you have a serialize and a deserialize function for it. 
```suggestion
    file.write(serializeCards(clean_cards))

def serializeCard(card):
    return str(card[0] + "-" + card[1] + ";")
def serializeCards(cards):
    serializedCards = map(serializeCard, cards)
    return ''.join(serializedCards)
    

```